### PR TITLE
feat(web): replace list of `BlockCard` by new `PaginatedTable` on the blocks view

### LINKED ISSUE #452: Use a table instead of cards to display elements in blocks page view
Reuse [`Table`](https://github.com/Blobscan/blobscan/blob/next/apps/web/src/components/Table/index.tsx) component for this

The table should have the following columns:

1. **Arrow icon**
 **Block Number**
2. **Timestamp**
3. **Slot**
4. **Txs**: Amount of transactions
5. **Blobs**: Amount of blobs
7. **Blob Gas Price**
8. **Blob Gas Used**: It should display the [`BlobGasUsageDisplay`](https://github.com/Blobscan/blobscan/blob/next/apps/web/src/components/Displays/BlobGasUsageDisplay.tsx) component below the gas used.

When clicking on the arrow icon a nested table containing the block's transactions and blobs  is displayed. It should have the following columns:

1. **Tx Hash**
2. **Blob Versioned Hash**
3. **Rollup**: This cell should contain the compacted rollup badge  

image

### PR DESCRIPTION
**Pull request by xFJA**
### Checklist

- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description

It resolves #452 

Migrate the current 'block view' using a block card list to the new `PaginatedTable` component

### Screenshots

image

### CODE REVIEW

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+              ),
+            },
+            {
+              item: timestamp.fromNow(),
+            },
+            {
+              item: (
+                <span className="text-contentSecondary-light dark:text-contentSecondary-dark">
+                  {slot}
```
Comment (PJColombo) [inline apps/web/src/components/PaginatedTable/helpers/blocks.tsx:118]: Let's turn this into a link similar to this [one](https://github.com/Blobscan/blobscan/blob/f631c7cd27b06a67ad2c529eba2ae33a8320dec7/apps/web/src/pages/block/%5Bid%5D.tsx#L92)

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                <span className="text-contentSecondary-light dark:text-contentSecondary-dark">
+                  {blobCount}
+                </span>
+              ),
+            },
+            {
+              item: <EtherUnitDisplay amount={blobGasPrice} />,
+            },
+            {
+              item: <BlobGasUsageDisplay blobGasUsed={blobGasUsed} />,
```
Comment (PJColombo) [inline apps/web/src/components/PaginatedTable/helpers/blocks.tsx:140]: The blob gas usage display looks a little bit too big. Let's reduce its size.

Maybe we can add a new prop `size` to the component and adjust it based on this [values](https://github.com/Blobscan/blobscan/blob/f631c7cd27b06a67ad2c529eba2ae33a8320dec7/apps/web/src/types/styles.ts#L1)

**Review by PJColombo — CHANGES_REQUESTED** [review]
Good work!

Left a few reviews.

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx
[the remark this answers is not in the document]
Comment (xFJA) [reply apps/web/src/components/PaginatedTable/helpers/blocks.tsx:1]: Done :+1:

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        transactionHash: transaction.hash,
+        blobVersionedHash: blob.versionedHash,
+      }))
+  );
+
+  const rows = transactionsCombinedWithInnerBlobs.map(
+    ({ transactionHash, blobVersionedHash }) => ({
+      cells: [
+        {
+          item: transactionHash,
```
[the remark this answers is not in the document]
Comment (xFJA) [reply apps/web/src/components/PaginatedTable/helpers/blocks.tsx]: Done :+1:

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        },
+        {
+          item: blobVersionedHash,
```
[the remark this answers is not in the document]
Comment (xFJA) [reply apps/web/src/components/PaginatedTable/helpers/blocks.tsx]: Done :+1:

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx (hunk 1 above)
Comment (xFJA) [reply apps/web/src/components/PaginatedTable/helpers/blocks.tsx]: Done :+1:

File: apps/web/src/components/PaginatedTable/helpers/blocks.tsx (hunk 2 above)
Comment (xFJA) [reply apps/web/src/components/PaginatedTable/helpers/blocks.tsx]: I added a `compact` prop that turns it into two lines. Also, the `percentage` and its text have been minimized to reduce the width

**Review by PJColombo — APPROVED** [review]
LGTM!

**Pull request merged**